### DISCUSSION TITLE
Github student benefit

### DISCUSSION THREAD

**Discussion by MenTo0rGood** [community/community#78511] [GitHub Education] [answered]
### Hi everyone,

Hi!

I got approved for the benefits, but then got rejected, saying: "The item you uploaded is insufficient to demonstrate your current academic status.
Please see suggestions.
Suggested proof to expedite your approval:

You are unlikely to be verified until you have completed your [GitHub billing information](https://github.com/settings/billing/payment_information) with your full name exactly as it appears in your academic affiliation document. You do not have to add a payment method. You may need to log out and log back in to GitHub before re-applying."

I updated my billing information and now, when I try te re-apply the continue button from the bottom of the page is grayed out. Anyone know what I can do? Thanks!

**Comment by erickdc7** [comment, marked as answer]
It seems you're encountering a common issue where the "Continue" button becomes grayed out during the re-application process for GitHub Education benefits. Here are a few steps you can try to resolve this issue:

[1](https://github.com/orgs/community/discussions/54851). **Clear Browser Cache and Cookies**: Sometimes, clearing your browser's cache and cookies can resolve issues with unresponsive buttons on websites.

[2](https://github.com/orgs/community/discussions/50540). **Use a Different Browser**: Some users have reported that switching to a different browser, such as Microsoft Edge, has allowed them to proceed with the application.

3. **Check for Missing Fields**: Ensure that all required fields in the application form are filled out. [Missing information might cause the button to remain disabled](https://github.com/orgs/community/discussions/54851).

4. **Private Browsing Mode**: Try using your browser's private or incognito mode to re-apply. This can sometimes bypass issues caused by browser extensions or cached data.

5. **Log Out and Log Back In**: As suggested, make sure to log out of GitHub and then log back in after updating your billing information. This can help refresh your session and potentially enable the button.

If you've tried these steps and the issue persists, you may want to reach out to GitHub Support for further assistance. They can provide more specific guidance based on your account details and the exact nature of the problem.

I hope this helps, and you're able to re-apply for the GitHub Education benefits successfully! 😊

**Reply by MenTo0rGood** [reply]
Thanks!

I noticed that I got accepted in the end, even if the button is still grayed and the status is saying rejected. I received the copilot, the thing I was most forward looking for. Thanks anyway!
